.github/workflows/main.yml

```diff
@@ -44,10 +44,10 @@ jobs:
     - run: cargo test --verbose --all-features
     - run: cargo test --verbose --features serde
     - run: cargo test --verbose --features std
-    - run: cargo test --verbose --features kv_unstable
```

Comment:
We could start using [cargo-hack](https://crates.io/crates/cargo-hack) for this at some point (for another pr).